src/test/java/org/prebid/server/validation/RequestValidatorTest.java

```diff
@@ -768,11 +765,12 @@ public void validateShouldReturnValidationMessageWhenCantParseTargetingPriceGran
     }
 
```

Comment:
There is no scenario when any non-leading range has negative increment. We test the only scenario when the first range has negative increment value.